spec/models/form1095_b_spec.rb

```diff
@@ -73,21 +73,12 @@
     let!(:multi_search_form_1) { create(:form1095_b, veteran_icn: '123456787654321') }
     let!(:multi_search_form_3) { create(:form1095_b, veteran_icn: '123456787654321', tax_year: 2019) }
 
```

Comment:
This is being changed because I changed the scope to return an array of form1095b objects instead of an array of hashes.